Dockerfile

```diff
@@ -1,4 +1,7 @@
-FROM onsdigital/docker-aws-apache-wsgi:python-3.4.2-onbuild
+FROM python:3.4-onbuild
```

Comment:
Maybe we do that as a separate card, as i think some of our code isn't compatible and would need a slight re-work